hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/connector/HazelcastWriters.java

```diff
@@ -62,7 +62,7 @@ private HazelcastWriters() {
             @Nonnull String name,
             @Nullable ClientConfig clientConfig
     ) {
-        return ProcessorMetaSupplier.of(new WriteMapP.Supplier<>(asXmlString(clientConfig), name));
+        return ProcessorMetaSupplier.of(1, new WriteMapP.Supplier<>(asXmlString(clientConfig), name));
```

Comment:
Javadoc needs updating: https://github.com/hazelcast/hazelcast-jet/blob/ab58c5085964a66538344f0287dbf1b59cb760e6/hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/Sources.java#L164-L165

Reply:
I wonder if we should even document parallelism here.